# German (Germany) translations for fb_vmware.
# Copyright (C) 2024 Frank Brehm, Berlin
# This file is distributed under the same license as the fb_vmware project.
# Frank Brehm <frank.brehm@pixelpark.com>, 2024.
#
msgid ""
msgstr ""
"Project-Id-Version: fb_vmware 1.1.2\n"
"Report-Msgid-Bugs-To: frank@brehm-online.com\n"
"POT-Creation-Date: 2024-07-10 14:21+0200\n"
"PO-Revision-Date: 2024-07-10 14:30+0200\n"
"Last-Translator: Frank Brehm <frank@brehm-online.com>\n"
"Language: de_DE\n"
"Language-Team: Frank Brehm <frank@brehm-online.com>\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.15.0\n"

#: bin/get-vsphere-host-list:65 bin/get-vsphere-storage-cluster-list:65 bin/get-vsphere-vm-info:65
#: bin/get-vsphere-vm-list:65
msgid ""
"{c}-Object:\n"
"{a}"
msgstr ""
"{c}-Objekt:\n"
"{a}"

#: lib/fb_vmware/about.py:283 lib/fb_vmware/cluster.py:368 lib/fb_vmware/controller.py:266
#: lib/fb_vmware/datastore.py:230 lib/fb_vmware/dc.py:183 lib/fb_vmware/disk.py:388
#: lib/fb_vmware/ds_cluster.py:129 lib/fb_vmware/ether.py:510 lib/fb_vmware/host.py:247
#: lib/fb_vmware/host.py:670 lib/fb_vmware/host_port_group.py:198 lib/fb_vmware/network.py:141
#: lib/fb_vmware/vm.py:640
msgid "The given parameter {p!r} on calling method {m}() has failing attributes"
msgstr "Dem übergebenen Parameter {p!r} beim Aufruf der Methode {m}() fehlen Attribute"

#: lib/fb_vmware/about.py:291 lib/fb_vmware/cluster.py:279 lib/fb_vmware/dc.py:191
#: lib/fb_vmware/disk.py:397
msgid "Parameter {t!r} must be a {e} object, a {v} object was given instead."
msgstr "Der Parameter {t!r} muss ein {e}-Objekt sein, statt dessen wurde ein {v}-Objekt übergeben."

#: lib/fb_vmware/about.py:304 lib/fb_vmware/cluster.py:304 lib/fb_vmware/controller.py:308
#: lib/fb_vmware/datastore.py:270 lib/fb_vmware/dc.py:219 lib/fb_vmware/disk.py:419
#: lib/fb_vmware/ds_cluster.py:155 lib/fb_vmware/ether.py:475 lib/fb_vmware/host.py:279
#: lib/fb_vmware/host.py:588 lib/fb_vmware/host_port_group.py:222 lib/fb_vmware/network.py:173
#: lib/fb_vmware/vm.py:509
msgid "Creating {} object from:"
msgstr "Erstelle ein {}-Objekt aus:"

#: lib/fb_vmware/about.py:340 lib/fb_vmware/controller.py:313 lib/fb_vmware/disk.py:424
#: lib/fb_vmware/ether.py:480 lib/fb_vmware/host_port_group.py:227 lib/fb_vmware/vm.py:590
msgid "Created {} object:"
msgstr "Erstelltes {}-Objekt:"

#: lib/fb_vmware/app/__init__.py:97
msgid "{what} of {app} ..."
msgstr "{what} von {app} …"

#: lib/fb_vmware/app/__init__.py:100
msgid "Did not found any configured Vsphere environments."
msgstr "Keine VSPhere-Umgebung in der Konfiguration gefunden."

#: lib/fb_vmware/app/__init__.py:108
msgid "Checking for configured VSPhere instance {!r} ..."
msgstr "Überprüfe die konfigurierte VSPhere-Instanz {!r} …"

#: lib/fb_vmware/app/__init__.py:112
msgid "VSPhere {!r} not found in list of configured VSPhere instances."
msgstr "VSPhere {!r} wurde nicht in der Liste der konfigurierten VSPhere-Instanzen gefunden."

#: lib/fb_vmware/app/__init__.py:132
msgid "Enter password for {n} VSPhere user {u!r} on host {h!r}:"
msgstr "Geben Sie das Passwort für den {n}-VSPhere-Nutzer {u!r} auf Host {h!r} ein:"

#: lib/fb_vmware/app/__init__.py:145
msgid "The VSPhere names from configuration, in which the VMs should be searched."
msgstr "Die Namen der VSPheres aus der Konfiguration, in denen die VMs gesucht werden sollen."

#: lib/fb_vmware/app/__init__.py:153
msgid "Got command line arguments:"
msgstr "Erhaltene Kommandozeilen-Argumente:"

#: lib/fb_vmware/app/__init__.py:159
msgid "Initializing VSphere handlers ..."
msgstr "Initialisiere VSphere-Handler …"

#: lib/fb_vmware/app/__init__.py:172
msgid "Initializing handler for VSPhere {!r} ..."
msgstr "Initialisiere Handler für VSPhere {!r} …"

#: lib/fb_vmware/app/__init__.py:185
msgid "Could not initialize {} object from:"
msgstr "Konnte {}-Objekt nicht initialisieren aus:"

#: lib/fb_vmware/app/__init__.py:193
msgid "Cleaning up ..."
msgstr "Aufräumen …"

#: lib/fb_vmware/app/__init__.py:196
msgid "Closing VSPhere object {!r} ..."
msgstr "Schließe VSphere-Objekt {!r} …"

#: lib/fb_vmware/app/get_host_list.py:57
msgid "Tries to get a list of all physical hosts in VMWare VSphere and print it out."
msgstr "Versucht eine Liste aller physischen Hosts in VMWare VSphere zu erhalten und gibt diese aus."

#: lib/fb_vmware/app/get_host_list.py:101 lib/fb_vmware/app/get_vm_list.py:130
msgid "Filter options"
msgstr "Filteroptionen"

#: lib/fb_vmware/app/get_host_list.py:106
msgid "for names of hosts"
msgstr "für Namen von Hosts"

#: lib/fb_vmware/app/get_host_list.py:107
msgid "A regular expression to filter the output list of hosts by their name (Default: {!r})."
msgstr ""
"Ein regulärer Ausdruck, um die Ausgabeliste der Hosts nach ihrem Namen zu filtern (Vorgabe: "
"{!r})."

#: lib/fb_vmware/app/get_host_list.py:115
msgid "Filter output for online hosts."
msgstr "Filtern der Ausgabeliste nach Hosts, die online sind."

#: lib/fb_vmware/app/get_host_list.py:119
msgid "Filter output for offline hosts and templates."
msgstr "Filtern der Ausgabeliste nach Hosts, die offline sind.."

#: lib/fb_vmware/app/get_host_list.py:122 lib/fb_vmware/app/get_storage_cluster_list.py:102
#: lib/fb_vmware/app/get_vm_list.py:176
msgid "Output options"
msgstr "Ausgabeoptionen"

#: lib/fb_vmware/app/get_host_list.py:126 lib/fb_vmware/app/get_storage_cluster_list.py:111
#: lib/fb_vmware/app/get_vm_list.py:185
msgid ""
"The keys for sorting the output. Available keys are: {avail}. The default sorting keys are: "
"{default}."
msgstr ""
"Die Sortierschlüssel für die Ausgabe. Verfügbare Schlüssel sind: {avail}. Die Vorgabe-"
"Sortierschlüssel sind: {default}."

#: lib/fb_vmware/app/get_host_list.py:141 lib/fb_vmware/app/get_vm_list.py:203
msgid "Regular expression for filtering: {!r}"
msgstr "Regulärer Ausdruck als Filter: {!r}"

#: lib/fb_vmware/app/get_host_list.py:144 lib/fb_vmware/app/get_vm_list.py:206
msgid "Got a {c} for pattern {p!r}: {e}"
msgstr "Ein {c} für das Suchmuster {p!r} erhalten: {e}"

#: lib/fb_vmware/app/get_host_list.py:154 lib/fb_vmware/app/get_storage_cluster_list.py:132
#: lib/fb_vmware/app/get_vm_info.py:82 lib/fb_vmware/app/get_vm_list.py:242
msgid "Starting {a!r}, version {v!r} ..."
msgstr "Starte {a!r}, Version {v!r} …"

#: lib/fb_vmware/app/get_host_list.py:175
msgid "Getting all VSPhere hosts ..."
msgstr "Ermittele alle VSPhere Hosts …"

#: lib/fb_vmware/app/get_host_list.py:188
msgid "First found host:"
msgstr "Erster gefundener Host:"

#: lib/fb_vmware/app/get_host_list.py:245
msgid "Connect state"
msgstr "Verbindungsstatus"

#: lib/fb_vmware/app/get_host_list.py:246
msgid "CPU cores/threads"
msgstr "CPU-Cores/-Threads"

#: lib/fb_vmware/app/get_host_list.py:247
msgid "Memory in GiB"
msgstr "Hauptspeicher in GiB"

#: lib/fb_vmware/app/get_host_list.py:248
msgid "Vendor"
msgstr "Hersteller"

#: lib/fb_vmware/app/get_host_list.py:249
msgid "Model"
msgstr "Modell"

#: lib/fb_vmware/app/get_host_list.py:250
msgid "Maintenance"
msgstr "Wartungsmodus"

#: lib/fb_vmware/app/get_host_list.py:251
msgid "Online"
msgstr "Online"

#: lib/fb_vmware/app/get_host_list.py:253
msgid "Power State"
msgstr "Netztatus"

#: lib/fb_vmware/app/get_host_list.py:254
msgid "OS Name"
msgstr "Betriebssystemname"

#: lib/fb_vmware/app/get_host_list.py:255
msgid "OS Version"
msgstr "Betriebssystem-Version"

#: lib/fb_vmware/app/get_host_list.py:282
msgid "Yes"
msgstr "Ja"

#: lib/fb_vmware/app/get_host_list.py:284
msgid "No"
msgstr "Nein"

#: lib/fb_vmware/app/get_host_list.py:307 lib/fb_vmware/app/get_storage_cluster_list.py:314
#: lib/fb_vmware/app/get_vm_list.py:345
msgid "Line template: {}"
msgstr "Zeilenvorlage: {}"

#: lib/fb_vmware/app/get_host_list.py:323
msgid "Found no VMWare hosts."
msgstr "Fand keine VMWare Hosts."

#: lib/fb_vmware/app/get_host_list.py:325
msgid "Found one VMWare host."
msgid_plural "Found {} VMWare hosts."
msgstr[0] "Fand einen VMWare Host."
msgstr[1] "Fand {} VMWare Hosts."

#: lib/fb_vmware/app/get_storage_cluster_list.py:59
msgid "Tries to get a list of all datastore clusters in VMWare VSphere and print it out."
msgstr "Versucht eine Liste aller Datastore-Cluster in VMWare VSphere zu erhalten und gibt diese aus."

#: lib/fb_vmware/app/get_storage_cluster_list.py:106
msgid "Don't print the totals of all storage clusters."
msgstr "Die Gesamt-Zahlen aller Datastore-Cluster nicht ausgeben."

#: lib/fb_vmware/app/get_storage_cluster_list.py:175
msgid "Getting all VSPhere storage clusters ..."
msgstr "Ermittele alle VSPhere Datastore-Cluster …"

#: lib/fb_vmware/app/get_storage_cluster_list.py:238 lib/fb_vmware/app/get_vm_info.py:213
msgid "Total"
msgstr "Insgesamt"

#: lib/fb_vmware/app/get_storage_cluster_list.py:279
msgid "Capacity in GB"
msgstr "Kapazität in GiB"

#: lib/fb_vmware/app/get_storage_cluster_list.py:280
msgid "Free space in GB"
msgstr "Freier Platz in GiB"

#: lib/fb_vmware/app/get_storage_cluster_list.py:281
msgid "Calculated usage in GB"
msgstr "Berechnete Verwendung in GiB"

#: lib/fb_vmware/app/get_storage_cluster_list.py:282
msgid "Usage in percent"
msgstr "Verwendung in Prozent"

#: lib/fb_vmware/app/get_storage_cluster_list.py:343
msgid "Found one VMWare storage cluster."
msgid_plural "Found {} VMWare storage clusters."
msgstr[0] "Fand einen VMWare Datastore-Cluster."
msgstr[1] "Fand {} VMWare Datastore-Clusters."

#: lib/fb_vmware/app/get_storage_cluster_list.py:346
msgid "No VMWare storage clusters found."
msgstr "Keine  VMWare Storage-Cluster gefunden."

#: lib/fb_vmware/app/get_vm_info.py:48
msgid "Tries to get information about the given virtual machines in VMWare VSphere and print it out."
msgstr ""
"Versucht, Informationen über Virtuelle Maschinen in VMWare VSphere zu ermitteln und gibt diese "
"aus."

#: lib/fb_vmware/app/get_vm_info.py:68
msgid "Names of the VM to get information."
msgstr "Die Namen der VMs, über die Informationen ermittelt werden sollen."

#: lib/fb_vmware/app/get_vm_info.py:122
msgid "NOT FOUND"
msgstr "NICHT GEFUNDEN"

#: lib/fb_vmware/app/get_vm_info.py:150 lib/fb_vmware/app/get_vm_info.py:176
#: lib/fb_vmware/app/get_vm_info.py:229
msgid "Unknown"
msgstr "Unbekannt"

#: lib/fb_vmware/app/get_vm_info.py:193 lib/fb_vmware/app/get_vm_info.py:220
msgid "None"
msgstr "Keine"

#: lib/fb_vmware/app/get_vm_info.py:202
msgid "Disk"
msgid_plural "Disks"
msgstr[0] "Platte"
msgstr[1] "Platten"

#: lib/fb_vmware/app/get_vm_info.py:244
msgid "Custom Value"
msgid_plural "Custom Values"
msgstr[0] "Benutzerdefinierter Wert"
msgstr[1] "Benutzerdefinierte Werte"

#: lib/fb_vmware/app/get_vm_info.py:265
msgid "Pulling full data of VM {!r} ..."
msgstr "Hole vollständige Daten der VM {!r} …"

#: lib/fb_vmware/app/get_vm_list.py:58
msgid "Tries to get a list of all virtual machines in VMWare VSphere and print it out."
msgstr "Versucht eine Liste aller virtuellen Maschinen in VMWare VSphere zu erhalten und gibt diese aus."

#: lib/fb_vmware/app/get_vm_list.py:135
msgid "for names of VMs"
msgstr "für Name von VMs"

#: lib/fb_vmware/app/get_vm_list.py:136
msgid "A regular expression to filter the output list of VMs by their name (Default: {!r})."
msgstr "Ein regulärer Ausdruck, um die Ausgabeliste der VMs nach ihrem Namen zu filtern(Vorgabe: {!r})."

#: lib/fb_vmware/app/get_vm_list.py:143
msgid "TYPE"
msgstr "TYP"

#: lib/fb_vmware/app/get_vm_list.py:144
msgid "Filter output for the type of the VM. Valid values are {li} (Default: {dflt!r})."
msgstr "Filtern der Ausgabeliste nach dem Typ. Gültige Werte sind {li} (Vorgabe: {dflt!r})."

#: lib/fb_vmware/app/get_vm_list.py:153
msgid "Filter output for online VMs."
msgstr "Filtern der Ausgabeliste nach eingeschalteten VMs."

#: lib/fb_vmware/app/get_vm_list.py:157
msgid "Filter output for offline VMs and templates."
msgstr "Filtern der Ausgabeliste nach ausgeschalteten VMs und Vorlagen."

#: lib/fb_vmware/app/get_vm_list.py:162
msgid "for VMWare hardware config version"
msgstr "für die VMWare-Hardwarekonfigurationsversion"

#: lib/fb_vmware/app/get_vm_list.py:163
msgid ""
"A regular expression to filter the output list of VMs by the VMWare hardware configuration "
"version (e.g. '{}')."
msgstr ""
"Ein regulärer Ausdruck, um die Ausgabeliste der VMs nach ihrer VMWare-"
"Hardwarekonfigurationsversion zu filtern (z.Bsp. '{}')."

#: lib/fb_vmware/app/get_vm_list.py:170
msgid "for the Operating System version"
msgstr "für die Betriebssystemversion"

#: lib/fb_vmware/app/get_vm_list.py:171
msgid ""
"A regular expression to filter the output list of VMs by their Operating System version, e.g. "
"'{}'."
msgstr ""
"Ein regulärer Ausdruck, um die Ausgabeliste der VMs nach ihrer VMWare-"
"Hardwarekonfigurationsversion zu filtern (z.Bsp. '{}')."

#: lib/fb_vmware/app/get_vm_list.py:180
msgid "Detailed output list (quering data needs some time longer)."
msgstr "Detaillierte Ausgabeliste (die Abfrage der Daten benötigt signifikant mehr Zeit)."

#: lib/fb_vmware/app/get_vm_list.py:213
msgid "Detailed output is required because of your given options."
msgstr "Eine detaillierte Ausgabe is wegen der von Ihnen gewählten Optionen notwendig."

#: lib/fb_vmware/app/get_vm_list.py:225
msgid "Sorting key {!r} not usable, if not detailed output was given."
msgstr "Der Sortierschlüssel {!r} ist nicht verwendbar, da keine detaillierte Ausgabe angezeigt wird."

#: lib/fb_vmware/app/get_vm_list.py:229
msgid "No usable sorting keys found, using default sorting keys {}."
msgstr "Keine verwendbaren Sortierschlüssel gefunden, verwende Vorgabe-Sortierschlüssel {}."

#: lib/fb_vmware/app/get_vm_list.py:265
msgid "Getting all VSPhere VMs ..."
msgstr "Ermittele alle VSPhere VMs …"

#: lib/fb_vmware/app/get_vm_list.py:274
msgid "Using sorting keys:"
msgstr "Verwendete Sortierschlüssel:"

#: lib/fb_vmware/app/get_vm_list.py:362
msgid "Found no VMWare VMs."
msgstr "Fand keine VMWare VMs."

#: lib/fb_vmware/app/get_vm_list.py:364
msgid "Found one VMWare VM."
msgid_plural "Found {} VMWare VMs."
msgstr[0] "Fand eine VMWare VM."
msgstr[1] "Fand {} VMWare VMs."

#: lib/fb_vmware/app/get_vm_list.py:431
msgid "Found a {} object:"
msgstr "Fand ein {}-Objekt:"

#: lib/fb_vmware/base.py:79
msgid "The given parameter {pc!r} ({pv!r}) is not a {o} object."
msgstr "Der übergebene Parameter {pc!r} ({pv!r}) ist kein {o}-Objekt."

#: lib/fb_vmware/base.py:84
msgid "No VSPhere host name or address given in {w}."
msgstr "Kein VSPhere-Hostname oder -Adresse übergeben in {w}."

#: lib/fb_vmware/base.py:88
msgid "The {c} object given as {w} is not initialized."
msgstr "Das als {w} übergebene {c}-Objekt ist nicht initialisiert."

#: lib/fb_vmware/base.py:184
msgid "Connecting to vSphere {!r} ..."
msgstr "Verbinden zu vSphere {!r} …"

#: lib/fb_vmware/base.py:214
msgid "Disconnecting from VSPhere {!r}."
msgstr "Trenne Verbindung zu VSPhere {!r}."

#: lib/fb_vmware/cluster.py:261 lib/fb_vmware/controller.py:188 lib/fb_vmware/datastore.py:356
#: lib/fb_vmware/dc.py:146 lib/fb_vmware/disk.py:275 lib/fb_vmware/ds_cluster.py:193
#: lib/fb_vmware/ether.py:315 lib/fb_vmware/host.py:546 lib/fb_vmware/host_port_group.py:161
#: lib/fb_vmware/network.py:212 lib/fb_vmware/obj.py:275 lib/fb_vmware/vm.py:469
msgid "Comparing {} objects ..."
msgstr "Vergleiche {}-Objekte …"

#: lib/fb_vmware/cluster.py:312
msgid "Cluster {c!r} has network {n!r}."
msgstr "Das Cluster {c!r} besitzt das Netzwerk {n!r}."

#: lib/fb_vmware/cluster.py:319
msgid "Cluster {c!r} has datastore {d!r}."
msgstr "as Cluster {c!r} besitzt den Datastore {d!r}."

#: lib/fb_vmware/cluster.py:351 lib/fb_vmware/cluster.py:358
msgid "The given parameter {p!r} on calling method {m}() is not a sequence type."
msgstr "Der übergebene Parameter {p!r} beim Aufruf der Methode {m}() ist kein sequenzieller Typ."

#: lib/fb_vmware/config/__init__.py:162
msgid "An empty name for a VSPhere datacenter is not allowed."
msgstr "Ein leerer Name für ein VSPhere-Datacenter ist nicht erlaubt."

#: lib/fb_vmware/config/__init__.py:176
msgid "An empty user name for connecting to a VSPhere datacenter is not allowed."
msgstr "Ein leerer Nutzername für die Verbindung zu einem VSPhere-Datacenter ist nicht erlaubt."

#: lib/fb_vmware/config/__init__.py:285
msgid "Unknown key {k!r} with value {v!r} for VSphere {vs!r} in section {sec!r} found."
msgstr "Unbekannter Schlüssel {k!r} mit dem Wert {v!r} für VSphere {vs!r} in Abschnitt {sec!r} gefunden."

#: lib/fb_vmware/config/__init__.py:291
msgid "{e} in section {sn!r} for VSphere {vs!r}:"
msgstr "{e} in Abschnitt {sn!r} für VSphere {vs!r}:"

#: lib/fb_vmware/config/__init__.py:297
msgid "There must be given at least the VSPhere hostname in section {sn!r} for VSphere {vs!r}."
msgstr "Es muss mindestens der VSPhere-Hostname in Abschnitt {sec!r} für VSphere {vs!r} angegeben werden."

#: lib/fb_vmware/config/__init__.py:439
msgid "Evaluating config section {!r}:"
msgstr "Untersuche Kofigurations-Abschnitt {!r}:"

#: lib/fb_vmware/config/__init__.py:448
msgid "Empty VSphere name found."
msgstr "Leeren VSphere-Namen gefunden."

#: lib/fb_vmware/connect.py:119
msgid "Trying to get some 'about' information from VSphere."
msgstr "Versuche einige 'about'-Informationen von VSphere zu erhalten."

#: lib/fb_vmware/connect.py:134
msgid "Got a {c} on requesting 'about' information from VSPhere {url}: {e}"
msgstr "Habe einen {c} beim Ermitteln der 'about'-Informationen von VSphere {url} erhalten: {e}"

#: lib/fb_vmware/connect.py:144
msgid "VSphere version: {!r}"
msgstr "VSphere-Version: {!r}"

#: lib/fb_vmware/connect.py:146
msgid "Found VSphere about-information:"
msgstr "Gefundene VSphere-'about'-Informationen:"

#: lib/fb_vmware/connect.py:151
msgid "Trying to get datacenter from VSphere ..."
msgstr "Versuche das datacenter von VSphere zu erhalten …"

#: lib/fb_vmware/connect.py:165
msgid "Found VSphere datacenter {!r}."
msgstr "Fand VSphere-Datacenter {!r}."

#: lib/fb_vmware/connect.py:167
msgid "Info about datacenter:"
msgstr "Informationen über das Datacenter:"

#: lib/fb_vmware/connect.py:178
msgid "Trying to get all clusters from VSphere ..."
msgstr "Versuche alle Cluster von VSphere zu erhalten …"

#: lib/fb_vmware/connect.py:202 lib/fb_vmware/connect.py:207
msgid "Found clusters:"
msgstr "Gefundene Cluster:"

#: lib/fb_vmware/connect.py:223 lib/fb_vmware/connect.py:491
msgid "Found standalone host"
msgstr "Fand einzelstehenden Host"

#: lib/fb_vmware/connect.py:225 lib/fb_vmware/connect.py:493
msgid "Found cluster"
msgstr "Gefundene Cluster"

#: lib/fb_vmware/connect.py:226 lib/fb_vmware/connect.py:494
msgid "host"
msgid_plural "hosts"
msgstr[0] "Host"
msgstr[1] "Hosts"

#: lib/fb_vmware/connect.py:227 lib/fb_vmware/connect.py:495
msgid "CPU"
msgid_plural "CPUs"
msgstr[0] "CPU"
msgstr[1] "CPUs"

#: lib/fb_vmware/connect.py:228 lib/fb_vmware/connect.py:496
msgid "thread"
msgid_plural "threads"
msgstr[0] "Thread"
msgstr[1] "Threads"

#: lib/fb_vmware/connect.py:229 lib/fb_vmware/connect.py:497
msgid "network"
msgid_plural "networks"
msgstr[0] "Netzwerk"
msgstr[1] "Netzwerke"

#: lib/fb_vmware/connect.py:230 lib/fb_vmware/connect.py:498
msgid "datastore"
msgid_plural "datastores"
msgstr[0] "Datastore"
msgstr[1] "Datastores"

#: lib/fb_vmware/connect.py:231 lib/fb_vmware/connect.py:499
msgid ""
"{on} {cl!r}, {h} {h_l}, {cpu} {cpu_l}, {thr} {t_l}, {mem:0.1f} GiB Memory, {net} {nw_l} and {ds}"
" {ds_l}."
msgstr ""
"{on} {cl!r}, {h} {h_l}, {cpu} {cpu_l}, {thr} {t_l}, {mem:0.1f} GiB Hauptspeicher, {net} {nw_l} "
"und {ds} {ds_l}."

#: lib/fb_vmware/connect.py:254
msgid "Trying to get all datastores from VSphere ..."
msgstr "Versuche alle Datastores von VSPhere zu erhalten …"

#: lib/fb_vmware/connect.py:277 lib/fb_vmware/connect.py:279
msgid "Found datastores:"
msgstr "Gefundene Datastores:"

#: lib/fb_vmware/connect.py:287
msgid "Datastore mappings:"
msgstr "Datastore-Mappings:"

#: lib/fb_vmware/connect.py:302
msgid "Datastore {!r} seems to be local."
msgstr "Der Datastore {!r} scheint lokal zu sein."

#: lib/fb_vmware/connect.py:308
msgid "Found datastore {ds!r} of type {t!r}, capacity {c:0.1f} GByte."
msgstr "Fand Datastore {ds!r} vom Typ {t!r} mit einer Kapazität von {c:0.1f} GiByte."

#: lib/fb_vmware/connect.py:317
msgid "Trying to get all datastore clusters from VSphere ..."
msgstr "Versuche alle Datastore-Cluster von VSPhere zu erhalten …"

#: lib/fb_vmware/app/get_storage_cluster_list.py:183 lib/fb_vmware/connect.py:341
#: lib/fb_vmware/connect.py:344
msgid "Found datastore clusters:"
msgstr "Gefundene Datastore-Cluster:"

#: lib/fb_vmware/connect.py:346
msgid "No VSphere datastore clusters found."
msgstr "Keine VSphere-Datastore-Cluster gefunden."

#: lib/fb_vmware/connect.py:352
msgid "Datastore cluster mappings:"
msgstr "Datastore-Cluster-Mappings:"

#: lib/fb_vmware/connect.py:358 lib/fb_vmware/connect.py:418
msgid "Found a {} child."
msgstr "Fand ein {}-Child."

#: lib/fb_vmware/connect.py:376
msgid "Trying to get all networks from VSphere ..."
msgstr "Versuche alle Netzwerke von VSphere zu bekommen."

#: lib/fb_vmware/connect.py:398
msgid "Found one VSphere network."
msgid_plural "Found {n} VSphere networks."
msgstr[0] "Fand ein VSphere-Netzwerk."
msgstr[1] "Fand {n} VSphere-Netzwerke."

#: lib/fb_vmware/connect.py:402 lib/fb_vmware/connect.py:404
msgid "Found VSphere networks:"
msgstr "Gefundene VSphere-Netzwerke:"

#: lib/fb_vmware/connect.py:406
msgid "No VSphere networks found."
msgstr "Keine VSphere-Netzwerke gefunden."

#: lib/fb_vmware/connect.py:412
msgid "Network mappings:"
msgstr "Netzwerk-Mappings:"

#: lib/fb_vmware/connect.py:438 lib/fb_vmware/connect.py:642
msgid "Parameter {p!r} => {r!r} seems not to be a regex object."
msgstr "Der Parameter {p!r} => {r!r} scheint kein Regex-Objekt zu sein."

#: lib/fb_vmware/connect.py:441
msgid "Trying to get all host systems from VSphere with name pattern {!r} ..."
msgstr "Versuche eine Liste aller Hostsysteme mit dem Namensmuster {!r} zu ermitteln …"

#: lib/fb_vmware/connect.py:445
msgid "Trying to get all host systems from VSphere ..."
msgstr "Versuche alle Host-Systeme von VSphere zu ermitteln …."

#: lib/fb_vmware/connect.py:472 lib/fb_vmware/connect.py:477
msgid "Found hosts:"
msgstr "Gefundene Hosts:"

#: lib/fb_vmware/connect.py:483
msgid "Checking {o}-object in cluster {c!r} ..."
msgstr "Überprüfe {o}-Objekt in Cluster {c!r} …"

#: lib/fb_vmware/connect.py:517
msgid "Found host {h!r} in cluster {c!r}."
msgstr "Fand Host {h!r} in Cluster {c!r}."

#: lib/fb_vmware/connect.py:532
msgid "Searching for VM {n!r} (pattern: {p!r}) in VSPhere {v!r} ..."
msgstr "Suche nach VM {n!r} (Suchmuster: {p!r}) in VSPhere {v!r} …"

#: lib/fb_vmware/connect.py:540
msgid "VSphere VM {!r} not found."
msgstr "Die VSphere-VM {!r} wurde nicht gefunden."

#: lib/fb_vmware/connect.py:553 lib/fb_vmware/datastore.py:239 lib/fb_vmware/ds_cluster.py:138
#: lib/fb_vmware/iface.py:59 lib/fb_vmware/network.py:149
msgid "Parameter {t!r} must be a {e}, {v!r} was given."
msgstr "Der Parameter {t!r} muss ein {e}-Objekt sein, statt dessen wurde ein {v}-Objekt übergeben."

#: lib/fb_vmware/connect.py:646
msgid "Parameter {p1!r} and {p2!r} may not be {w!r} at the same time."
msgstr "Die Parameter {p1!r} und {p2!r} dürfen nicht {w!r} zur selben Zeit sein."

#: lib/fb_vmware/connect.py:650
msgid "Trying to get list of VMs with name pattern {!r} ..."
msgstr "Versuche eine Liste der VMs mit dem Namensmuster {!r} zu ermitteln …"

#: lib/fb_vmware/connect.py:670
msgid "Searching in path {!r} ..."
msgstr "Suche in Pfad {!r} …"

#: lib/fb_vmware/connect.py:683
msgid "Found one VM with pattern {p!r}."
msgid_plural "Found {no} VMs with pattern {p!r}."
msgstr[0] "Fand eine VM mit dem Suchmuster {p!r}."
msgstr[1] "Fand {no} VMs mit dem Suchmuster {p!r}."

#: lib/fb_vmware/connect.py:718
msgid "Checking VM {!r} ..."
msgstr "Überprüfe VM {!r} …"

#: lib/fb_vmware/connect.py:721
msgid "Checking VM {!r} for being a template ..."
msgstr "Überprüfe, ob die VM {!r} eine Vorlage ist …"

#: lib/fb_vmware/connect.py:723
msgid "Checking VM {!r} for being not a template ..."
msgstr "Überprüfe, ob die VM {!r} keine Vorlage ist …"

#: lib/fb_vmware/connect.py:731
msgid "Checking VM {!r} for pattern."
msgstr "Überprüfe VM {!r} nach Muster."

#: lib/fb_vmware/connect.py:734
msgid "Found VM {!r}."
msgstr "Habe VM {!r} gefunden."

#: lib/fb_vmware/connect.py:798
msgid "VM {!r} is already powered on."
msgstr "Die VM {!r} ist bereits eingeschaltet."

#: lib/fb_vmware/connect.py:801
msgid "Powering on VM {!r} ..."
msgstr "Schalte VM {!r} ein …"

#: lib/fb_vmware/connect.py:805
msgid "VM {!r} successful powered on."
msgstr "Die VM {!r} wurde erfolgreich eingeschaltet."

#: lib/fb_vmware/connect.py:829
msgid "VM {!r} is already powered off."
msgstr "Die VM {!r} ist bereits ausgeschaltet."

#: lib/fb_vmware/connect.py:832
msgid "Powering off VM {!r} ..."
msgstr "Schalte VM {!r} aus …"

#: lib/fb_vmware/connect.py:836
msgid "VM {!r} successful powered off."
msgstr "Die VM {!r} wurde erfolgreich ausgeschaltet."

#: lib/fb_vmware/connect.py:845
msgid "Ensuring existence of VSphere VM folders:"
msgstr "Stelle Existenz von VSphere VM-Ordnern sicher:"

#: lib/fb_vmware/connect.py:862
msgid "Trying to get VM folder object for path {!r}."
msgstr "Versuche das VM-Ordnerobjekt für den Pfad {!r} zu erhalten."

#: lib/fb_vmware/connect.py:887
msgid "Checking single VM folder {i}: {f!r}."
msgstr "Überprüfe einfachen VM-Ordner {i}: {f!r}."

#: lib/fb_vmware/connect.py:898
msgid "Found VM folder {n}, parent: {p}"
msgstr "Habe VM-Ordner {n} gefunden, Eltern-Ordner: {p}"

#: lib/fb_vmware/connect.py:916
msgid "Ensuring existence of VSphere VM folder {!r}."
msgstr "Stelle Existenz des VSphere-VM-Ordners {!r} sicher."

#: lib/fb_vmware/connect.py:941
msgid "VM Folder {!r} already exists."
msgstr "Der VM-Ordner {!r} existiert bereits."

#: lib/fb_vmware/connect.py:943
msgid "Creating VM folder {!r} ..."
msgstr "Erstelle VM-Ordner {!r} …"

#: lib/fb_vmware/connect.py:945
msgid "Simulation mode, don't creating it."
msgstr "Simulationsmodus, erstelle ihn nicht."

#: lib/fb_vmware/connect.py:961
msgid "Waiting for tasks to finish ..."
msgstr "Warte auf Beendigung der Tasks …"

#: lib/fb_vmware/connect.py:973
msgid "Waiting at most {m} seconds for tasks {t} to finish ..."
msgstr "Warte maximal {m} Sekunden auf die Beendigung der Tasks {t} …"

#: lib/fb_vmware/connect.py:976
msgid "Waiting for tasks {} to finish ..."
msgstr "Warte auf Beendigung der Tasks {} …"

#: lib/fb_vmware/connect.py:996
msgid "Waiting ..."
msgstr "Warten …"

#: lib/fb_vmware/connect.py:1030
msgid "Creating VM {!r} ..."
msgstr "Erstelle VM {!r} …"

#: lib/fb_vmware/connect.py:1033
msgid "Simulation mode - VM {!r} will not be created."
msgstr "Simulations-Modus - die VM {!r} wird nicht erstellt."

#: lib/fb_vmware/connect.py:1052
msgid "Generating create spec for VM {!r} ..."
msgstr "Generiere Erstellungs-Spec für VM {!r} …"

#: lib/fb_vmware/connect.py:1060
msgid "Datastore path: {!r}"
msgstr "Datastore-Pfad: {!r}"

#: lib/fb_vmware/connect.py:1063
msgid "VM path name: {!r}"
msgstr "VM-Pfadname: {!r}"

#: lib/fb_vmware/connect.py:1120
msgid "Generated VM config:"
msgstr "Generierte VM-Konfiguration:"

#: lib/fb_vmware/connect.py:1129
msgid "Given disksize {!r} must be greater than zero."
msgstr "Die gegebene Plattengröße muss größer als Null sein."

#: lib/fb_vmware/connect.py:1142
msgid "There may be created at most 6 disks, but {} were given."
msgstr "Es können höchstens 6 Platten erstellt werden, aber {} wurden übergeben."

#: lib/fb_vmware/connect.py:1156
msgid "Generating spec for SCSI controller and one disk: {d}"
msgid_plural "Generating spec for SCSI controller and {n} disks: {d}"
msgstr[0] "Generiere Spec für SCSI-Controller und eine Disk: {d}"
msgstr[1] "Generiere Spec für SCSI-Controller und {n} Disks: {d}"

#: lib/fb_vmware/connect.py:1159
msgid "Generating spec for SCSI controller without disks."
msgstr "Generiere Spec für SCSI-Controller ohne Disks."

#: lib/fb_vmware/connect.py:1184
msgid "Adding spec for disk {n!r} with {gb} GiB => {kb} KiByte."
msgstr "Füge Spec für Disk {n!r} mit {gb} GiB => {kb} KiByte hinzu."

#: lib/fb_vmware/connect.py:1219
msgid "Invalid Interface description {!r} given."
msgstr "Ungültige Interface-Beschreibung {!r} übergeben."

#: lib/fb_vmware/connect.py:1230
msgid "Defined interface:"
msgstr "Definierte Netzwerkschnittstelle:"

#: lib/fb_vmware/connect.py:1234
msgid "Adding spec for network interface {d!r} (Network {n!r}, MAC: {m!r}, summary: {s!r})."
msgstr ""
"Füge Spec für Netzwerkschnittstelle {d!r} (Netzwerk {n!r}, MAC: {m!r}, Zusammenfassung: {s!r}) "
"hinzu."

#: lib/fb_vmware/connect.py:1286
msgid "Purging VM {!r} ..."
msgstr "Lösche VM {!r} vollständig …"

#: lib/fb_vmware/connect.py:1290
msgid "VM {!r} successful removed."
msgstr "VM {!r} wurde erfolgreich gelöscht."

#: lib/fb_vmware/connect.py:1321
msgid "Did not found virtual ethernet device No. {no} (found {count} devices)."
msgstr "Virtuelles Ethernet-Device Nummer {no} wurde nicht gefunden (es wurden {count} Devices gefunden)."

#: lib/fb_vmware/connect.py:1340
msgid "Changes of MAC address:"
msgstr "Ändern von MAC-Adresse:"

#: lib/fb_vmware/connect.py:1344
msgid "Successful changed MAC address of VM {v!r} to {m!r}."
msgstr "Erfolgreich MAC-Adresse der VM {v!r} auf {m!r} geändert."

#: lib/fb_vmware/connect.py:1359
msgid "Trying to detect all field names of custom field definitions."
msgstr "Versuche alle Namen von benutzerdefinierten Felddefinitionen zu ermitteln."

#: lib/fb_vmware/connect.py:1376
msgid "Got a {c} on requesting custom field names from VSPhere {url}: {e}"
msgstr "Habe einen {c} beim Ermitteln benutzerdefinierter Feldnamen von VSphere {url} erhalten: {e}"

#: lib/fb_vmware/connect.py:1382
msgid "Got custom field names from VSPhere {}:"
msgstr "Erhaltene benutzerdefinierte Feldnamenvon VSphere {}:"

#: lib/fb_vmware/controller.py:57
msgid "Virtual IDE controller"
msgstr "Virtueller IDE-Kontroller"

#: lib/fb_vmware/controller.py:58
msgid "Virtual VNME controller"
msgstr "Virtueller VNME-Kontroller"

#: lib/fb_vmware/controller.py:59
msgid "Virtual PCI controller"
msgstr "Virtueller PCO-Kontroller"

#: lib/fb_vmware/controller.py:60
msgid "Virtual controller for keyboards and mice"
msgstr "Virtueller Kontroller für Tastaturen und Mäuse"

#: lib/fb_vmware/controller.py:61
msgid "Virtual AHCI SATA controller"
msgstr "Virtueller AHCI-SATA-Kontroller"

#: lib/fb_vmware/controller.py:62
msgid "Virtual SATA controller"
msgstr "Virtueller SATA-Kontroller"

#: lib/fb_vmware/controller.py:63
msgid "Virtual paravirtualized SCSI controller"
msgstr "Virtueller paravirtualisierter SCSI-Kontroller"

#: lib/fb_vmware/controller.py:64
msgid "Virtual BusLogic SCSI controller"
msgstr "Virtueller BusLogic SCSI-Kontroller"

#: lib/fb_vmware/controller.py:65
msgid "Virtual LSI SCSI controller"
msgstr "Virtueller LSI SCSI-Kontroller"

#: lib/fb_vmware/controller.py:66
msgid "Virtual LSI Logic SAS SCSI controller"
msgstr "Virtueller LSI Logic Logic SCSI-Kontroller"

#: lib/fb_vmware/controller.py:67
msgid "Virtual SCSI controller"
msgstr "Virtueller SCSI-Kontroller"

#: lib/fb_vmware/controller.py:68
msgid "Virtual Super IO Controller for floppy drives, parallel ports, and serial ports"
msgstr "Virtueller Super-IO-Kontroller für Diskettenlaufwerke, Parallel- und serielle Ports"

#: lib/fb_vmware/controller.py:70
msgid "Virtual USB controller (USB 1.1 and 2.0)"
msgstr "Virtueller USB-Kontroller (USB 1.1 and 2.0)"

#: lib/fb_vmware/controller.py:71
msgid "Virtual USB Extensible Host Controller Interface (USB 3.0)"
msgstr "Virtueller USB Extensible Host Kontroller Interface (USB 3.0)"

#: lib/fb_vmware/controller.py:72
msgid "Unknown virtual controller"
msgstr "Unbekannter virtueller Kontroller"

#: lib/fb_vmware/controller.py:275 lib/fb_vmware/ether.py:424 lib/fb_vmware/host.py:255
#: lib/fb_vmware/host.py:568 lib/fb_vmware/host_port_group.py:206 lib/fb_vmware/vm.py:493
msgid "Parameter {t!r} must be a {e}, {v!r} ({vt}) was given."
msgstr "Der Parameter {t!r} muss ein {e}-Objekt sein, statt dessen wurde {v!r} ({vt}) übergeben."

#: lib/fb_vmware/controller.py:297
msgid "Checking class of controller: {!r}"
msgstr "Überprüfe die Klasse des Kontrollers: {!r}"

#: lib/fb_vmware/controller.py:322 lib/fb_vmware/disk.py:433 lib/fb_vmware/ether.py:521
#: lib/fb_vmware/host_port_group.py:236 lib/fb_vmware/vm.py:651
msgid "Invalid type {t!r} as an item of a {c}, only {o} objects are allowed."
msgstr "Ungültiger Typ {t!r} als ein Eintrag von {c}, ausschließlich {o}-Objekte sind erlaubt."

#: lib/fb_vmware/controller.py:389 lib/fb_vmware/disk.py:500 lib/fb_vmware/ether.py:588
#: lib/fb_vmware/host.py:792 lib/fb_vmware/host_port_group.py:324 lib/fb_vmware/vm.py:718
msgid "{m} takes at most {max} arguments ({n} given)."
msgstr "{m} akzeptiert höchstens {max} Argumente ({n} wurden gegeben)."

#: lib/fb_vmware/controller.py:428
msgid "Controller is not in controller list."
msgstr "Der Kontroller ist nicht in der Liste der Kontroller."

#: lib/fb_vmware/datastore.py:375 lib/fb_vmware/ds_cluster.py:212 lib/fb_vmware/network.py:231
msgid "Invalid item type {{!r}} to set, only {} allowed."
msgstr "Ungültiger Elementen-Typ {{!r}} beim Setzen, nur {} ist erlaubt."

#: lib/fb_vmware/datastore.py:377
msgid "The key {k!r} must be equal to the datastore name {n!r}."
msgstr "Der Schlüssel {k!r} muss identisch mit dem Namen des Datastores {n!r} sein."

#: lib/fb_vmware/datastore.py:378 lib/fb_vmware/ds_cluster.py:215 lib/fb_vmware/network.py:234
msgid "None type as key is not allowed."
msgstr "Ein None-Typ ist als Schlüssel nicht erlaubt."

#: lib/fb_vmware/datastore.py:379 lib/fb_vmware/ds_cluster.py:216 lib/fb_vmware/network.py:235
msgid "Empty key {!r} is not allowed."
msgstr "Ein leerer Schlüssel {!r} ist nicht erlaubt."

#: lib/fb_vmware/datastore.py:380 lib/fb_vmware/ds_cluster.py:217 lib/fb_vmware/network.py:236
#: lib/fb_vmware/obj.py:257 lib/fb_vmware/obj.py:266 lib/fb_vmware/obj.py:286 lib/fb_vmware/obj.py:298
msgid "Object {{!r}} is not a {} object."
msgstr "Das Objekt {{!r}} ist kein {}-Objekt."

#: lib/fb_vmware/datastore.py:623
msgid "Could not handle datastore type {!r}."
msgstr "Kann den Datastore-Typ {!r} nicht behandeln."

#: lib/fb_vmware/datastore.py:630
msgid "Could not found a datastore for {c:0.1f} GiB of type {t!r}."
msgstr "Konnte keinen Datastore für {c:0.1f} GiB vom Typ {t!r} finden."

#: lib/fb_vmware/datastore.py:637
msgid "Searching datastore for {c:0.1f} GiB of type {t!r}."
msgstr "Suche Datastore für {c:0.1f} GiB vom Typ {t!r}."

#: lib/fb_vmware/disk.py:540
msgid "Disk is not in disk list."
msgstr "Die Platte ist nicht in der Liste der Disks."

#: lib/fb_vmware/ds_cluster.py:214
msgid "The key {k!r} must be equal to the datastore cluster name {n!r}."
msgstr "Der Schlüssel {k!r} muss gleich dem Namen des Datastore-Clusters {n!r} sein."

#: lib/fb_vmware/errors.py:55
msgid "No VSphere datastores found."
msgstr "Keine VSphere-Datastores gefunden."

#: lib/fb_vmware/errors.py:90
msgid "Invalid name {n!r} for a {o} VSphere object."
msgstr "Ungültiger Name für ein {o}-VSphere-Objekt."

#: lib/fb_vmware/errors.py:93
msgid "Invalid name {!r} for a VSphere object."
msgstr "Ungültiger Name für ein VSphere-Objekt."

#: lib/fb_vmware/errors.py:110
msgid "The VSphere datacenter {!r} is not existing."
msgstr "Das VSphere-Datacenter {!r} existiert nicht."

#: lib/fb_vmware/errors.py:126
msgid "The VSphere Virtual machine {!r} was not found."
msgstr "Die VSphere-VM {!r} wurde nicht gefunden."

#: lib/fb_vmware/errors.py:145
msgid "No SAN based datastore found with at least {m:0.0f} MiB == {g:0.1f} GiB available space found."
msgstr ""
"Kein SAN-basierter Datastore mit wenigstens {m:0.0f} MiB == {g:0.1f} GiB verfügbarem Platz "
"gefunden."

#: lib/fb_vmware/errors.py:163
msgid "The network {!r} is not existing."
msgstr "Das Netzwerk {!r} existiert nicht."

#: lib/fb_vmware/errors.py:179
msgid "Could not connect to the vSphere {!r}."
msgstr "Konnte mich nicht zu VSPhere {!r} verbinden."

#: lib/fb_vmware/errors.py:195
msgid "Got a {c} on connecting to vSphere {url!r}:"
msgstr "{c} beim Verbinden zu VSPhere {url!r} erhalten:"

#: lib/fb_vmware/errors.py:219
msgid "Timeout on creating VM {vm!r} after {to:0.1f} seconds."
msgstr "Zeitüberschreitung bei der Erstellung der VM {vm!r} nach {to:0.1f} Sekunden."

#: lib/fb_vmware/errors.py:222
msgid "Timeout on creating VM {!r}."
msgstr "Zeitüberschreitung bei der Erstellung der VM {!r}."

#: lib/fb_vmware/errors.py:239
msgid "Invalid type of {!r} for a port of a VSPhere server"
msgstr "Ungültiger Typ von {!r} für einen Port eines VSPhere-Servers"

#: lib/fb_vmware/errors.py:264
msgid ""
"Invalid port number {port!r} for the VSphere server, PORT must be greater than zero and less or "
"equal to {max}."
msgstr ""
"Ungültige Portnummer {port!r} für den VSphere-Server, diese muss größer als Null und kleiner "
"oder gleich {max} sein."

#: lib/fb_vmware/ether.py:452
msgid "Checking class of ethernet card: {!r}"
msgstr "Überprüfe die Klasse der Ethernet-Karte: {!r}"

#: lib/fb_vmware/ether.py:627
msgid "card is not in card list."
msgstr "Die Karte ist nicht in der Liste der Ethernet-Karten."

#: lib/fb_vmware/host.py:346 lib/fb_vmware/vm.py:110
msgid "The name of the vsphere may not be empty."
msgstr "Der Name des VSPhere darf nicht leer sein."

#: lib/fb_vmware/host.py:573
msgid "Host {!r} seems to be offline!"
msgstr "Der Host {!r} scheint offline zu sein!"

#: lib/fb_vmware/host.py:723
msgid "Invalid type {{t!r}} as an item of a {{c}}, only {} objects are allowed."
msgstr "Ungültiger Typ {{t!r}} als ein Eintrag von {{c}}, ausschließlich {}-Objekte sind erlaubt."

#: lib/fb_vmware/host.py:831
msgid "host is not in host list."
msgstr "der Host ist nicht in der Liste der Hosts."

#: lib/fb_vmware/host_port_group.py:363
msgid "group is not in group list."
msgstr "Gruppe ist nicht in der Gruppenliste."

#: lib/fb_vmware/iface.py:127
msgid "Invalid MAC address {!r} for interface given."
msgstr "Ungültige IP-Adresse {!r} für Interface angegeben."

#: lib/fb_vmware/network.py:72
msgid "Trying to get IPv4 network {n!r} -> {i!r}."
msgstr "Versuche ein IPv6-Netzwerk {n!r} -> {i!r} zu bekommen."

#: lib/fb_vmware/network.py:79
msgid "Could not get IP network from network name {!r}."
msgstr "Konnte kein IP-Netzwerk aus dem Netzwerk-Namen {!r} ermitteln."

#: lib/fb_vmware/network.py:82
msgid "Network {!r} has no IP network assigned."
msgstr "Dem Netzwerk {!r} wurde kein IP-Netzwerk zugeordnet."

#: lib/fb_vmware/network.py:88
msgid "Initialized network object:"
msgstr "Initialisiertes Netzwerk-Objekt:"

#: lib/fb_vmware/network.py:233
msgid "The key {k!r} must be equal to the network name {n!r}."
msgstr "Der Schlüssel {k!r} muss identisch mit dem Namen des Netzwerkes {n!r} sein."

#: lib/fb_vmware/network.py:501
msgid "Searching VSphere network for address {} ..."
msgstr "Suche VSphere-Netzwerk für Adresse {} …"

#: lib/fb_vmware/network.py:507
msgid "Found network {n!r} for IP {i}."
msgstr "Fand Netzwerk {n!r} für IP {i}."

#: lib/fb_vmware/network.py:511
msgid "Could not find VSphere network for IP {}."
msgstr "Konnte kein VSphere-Netzwerk für die IP {} finden."

#: lib/fb_vmware/network.py:514
msgid "Could not find VSphere network for IP addresses {}."
msgstr "Konnte kein VSphere-Netzwerk für die IP-Adressen {} finden."

#: lib/fb_vmware/obj.py:84
msgid "The type of a {} may not be None."
msgstr "Der Typ von einem {} darf nicht None sein."

#: lib/fb_vmware/obj.py:89 lib/fb_vmware/obj.py:112 lib/fb_vmware/obj.py:135
msgid "Invalid {w}.{p} {v!r}."
msgstr "{w}.{p} ist ungültig: {v!r}."

#: lib/fb_vmware/obj.py:151
msgid "The name prefix of a {} may not be None."
msgstr "Der Namenspräfix eines {} darf nicht None sein."

#: lib/fb_vmware/obj.py:155
msgid "Invalid name prefix {p!r} for a {o}."
msgstr "Ungültiger Namenspräfix {p!r} für ein {o}."

#: lib/fb_vmware/vm.py:582
msgid "Unknown hardware device of type {}."
msgstr "Unbekanntes Hardware-Gerät vom Typ {}."

#: lib/fb_vmware/vm.py:585
msgid "There is something wrong wit VM {n!r} in cluster {c!r} and path {p!r} ..."
msgstr "Da ist irgendwas falsch mit der VM {n!r} in Cluster {c!r} und Pfad {p!r} …"

#: lib/fb_vmware/vm.py:757
msgid "VM is not in VM list."
msgstr "Die VM ist nicht in der Liste der Virtuellen Maschinen."

#: lib/fb_vmware/xlate.py:119
msgid "Module directory:"
msgstr "Modul-Verzeichnis:"

#: lib/fb_vmware/xlate.py:120
msgid "Base directory:"
msgstr "Basis-Verzeichnis:"

#: lib/fb_vmware/xlate.py:121
msgid "Locale directory:"
msgstr "Verzeichnis für Locales:"

#: lib/fb_vmware/xlate.py:122
msgid "Locale domain:"
msgstr "Locale-Domäne:"

#: lib/fb_vmware/xlate.py:123
msgid "Found .mo-file:"
msgstr "Gefundene .mo-Datei:"

